elm-app/src/elm/Utils/Html.elm

```diff
@@ -25,6 +27,47 @@ emptyNode =
     text ""
 
 
+{-| Splits a list into sub lists containing a given number of children.
+-}
+split : Int -> List a -> List (List a)
+split numberOfChildren list =
+    case take numberOfChildren list of
+        [] ->
+            []
+
+        listHead ->
+            listHead :: split numberOfChildren (drop numberOfChildren list)
+
+
+{-| Render the column with a given class.
+-}
+renderColumn : String -> Html msg -> Html msg
+renderColumn columnClass column =
+    div [ class columnClass ] [ column ]
+
+
+{-| Render the row with a list of columns.
+-}
+renderRow : String -> List (Html msg) -> Html msg
+renderRow columnClass row =
+    div [ class "row" ] (List.map (renderColumn columnClass) <| row)
+
+
+{-| Renders rows to a given number of columns. Gives the ability to wrap a list og
+Html msg with columns and rows:
+
+    renderBootstrapGrid 2 "col-md-6" List (div [] [], div [] [])
+
+-}
+renderBootstrapGrid : Int -> String -> List (Html msg) -> Html msg
```

Comment:
@amitaibu @noamoss This is the significant change that we need to add to contacts as well when approved.